fast64_internal/oot/skeleton/importer/functions.py

```diff
@@ -324,3 +324,20 @@ def ootImportSkeletonC(basePath: str, importSettings: OOTSkeletonImportSettings)
         applySkeletonRestPose(restPoseData, armatureObj)
         if isLOD:
             applySkeletonRestPose(restPoseData, LODArmatureObj)
+
+
+    if importAnimations:
+
+        if armatureObj:
```

Comment:
```suggestion
        if armatureObj is not None:
```
I think this is the intent here?